packages/logger/src/index.js

```diff
@@ -14,7 +14,8 @@ Object.assign(logger, {
   connect: (...args) => new Logger().connect(...args),
   remote: (...args) => new Logger().remote(...args),
   loglevel(level, flags = {}) {
-    if (flags.verbose) level = 'debug';
+    if (flags.debug) level = 'debug';
+    else if (flags.verbose) level = 'debug';
```

Comment:
Kept these as separate checks in case we ever decide that one of these should produce even more logs